pom.xml

```
@@ -76,8 +76,8 @@
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.319.x</artifactId>
-        <version>1508.v4b_d09ff0e893</version>
+        <artifactId>bom-2.361.x</artifactId>
+        <version>1763.v092b_8980a_f5e</version>
```

do you really need this to be this new?  have you checked in our internal repo that all the dependencies are include to avoid yet another a big bang approach?

whilst the BOM and PCT is not perfect - you should not just choose the latest for no reason.  We will be working on fixing that - so its better to choose the minimum version you need to work.  and do not merge bom updates from dependabot but take action if they fail.

hint - this will be blocked on at least an update of the credentials plugin to `1189.vf61b_a_5e2f62e`.

---

untested minimum version for this line is [`1580.v47b_429a_c853a`](https://github.com/jenkinsci/bom/releases/tag/1580.v47b_429a_c853a)

```suggestion
        <version>1580.v47b_429a_c853</version>
```

---

> hint - this will be blocked on at least an update of the credentials plugin to `1189.vf61b_a_5e2f62e`.

https://github.com/jenkinsci/bom/pull/1564 is in https://github.com/jenkinsci/bom/releases/tag/1706.vc166d5f429f8 but whatever

---

I was meaning into our (cloudbees) product.

but the credential version is 1214.xxx so no idea what I was looking at there.